src/server/api/controllers/users-controller.js

```diff
@@ -0,0 +1,28 @@
+const knex = require('../../config/db');
+
+const getUsers = async () => {
+  try {
+    const userList = await knex('users')
+      .select('*')
+      .limit(20);
+    return userList.length > 0 ? userList : 'No Users';
+  } catch (error) {
+    return error.message;
+  }
+};
```

Comment:
One thing I think could be nice to do instead of hardcoding `20` as the limit, what we could do it making a default value which can be overwritten in case we want to make the limit something else (more flexible). It would look something like this:

```javascript
const getUsers = async ({ limit = 20 }) => {
  try {
    const userList = await knex('users')
      .select('*')
      .limit(limit);
    return userList.length > 0 ? userList : 'No Users';
  } catch (error) {
    return error.message;
  }
};
```